.circleci/config.yml

```diff
@@ -1,5 +1,47 @@
 version: 2
 jobs:
+  sync_bot:
+    docker:
+      - image: ruby:2.3.6
+        environment:
+          BUNDLE_JOBS: 3
+          BUNDLE_RETRY: 3
+          BUNDLE_PATH: vendor/bundle
+          RAILS_ENV: develop
+    steps:
+      - checkout
+      - run:
+          name: install dockerize
```

Comment:
It doesn't look like `sync_bot` actually use the `dockerize` command, am I missing something?

Reply:
probably not. just copypasta :)